smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/serde/readwrite/DocumentWritingClosureUtils.kt

```diff
@@ -12,25 +9,11 @@ import software.amazon.smithy.swift.codegen.SwiftDependency
 import software.amazon.smithy.swift.codegen.SwiftWriter
 import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
 import software.amazon.smithy.swift.codegen.integration.serde.xml.NodeInfoUtils
-import software.amazon.smithy.swift.codegen.model.hasTrait
 
 class DocumentWritingClosureUtils(
```

Comment:
Refactored this existing type with the new utilities on service shapes for getting wire protocol.